sys/net/crosslayer/ng_pktbuf_static/ng_pktbuf_static.c

```diff
@@ -429,6 +429,8 @@ static inline bool _too_small_hole(_unused_t *a, _unused_t *b)
 
 static inline _unused_t *_merge(_unused_t *a, _unused_t *b)
 {
+    assert(b->next != NULL);
```

Comment:
Finding by `make scan-build`. It could hypothetically happen if `_too_small_hole()` returns true and `_next_unused` is still NULL. See conmit message.